target/linux/ramips/patches-6.6/802-GPIO-MIPS-ralink-add-gpio-driver-for-ralink-SoC.patch

```diff
@@ -363,12 +360,6 @@ Cc: linux-gpio@vger.kernel.org
 +		return -EINVAL;
 +	}
 +
-+	gpiobase = of_get_property(np, "ralink,gpio-base", NULL);
-+	if (gpiobase)
-+		rg->chip.base = be32_to_cpu(*gpiobase);
-+	else
-+		rg->chip.base = -1;
```

Comment:
Line 370 should remain because after removing it rg->chip.base is not initialized which causes incorrect behavior for the 6.6.36 kernel.
After leaving it, the GPIOs load correctly:
```
[    0.435617] rt2880_gpio 10000600.gpio: registering 24 gpios
[    0.441052] rt2880_gpio 10000600.gpio: registering 24 irq handlers
[    0.447495] rt2880_gpio 10000638.gpio: registering 16 gpios
[    0.452902] rt2880_gpio 10000638.gpio: registering 16 irq handlers
[    0.459245] rt2880_gpio 10000660.gpio: registering 32 gpios
[    0.464705] rt2880_gpio 10000660.gpio: registering 32 irq handlers
[    0.471145] rt2880_gpio 10000688.gpio: registering 1 gpios
[    0.476469] rt2880_gpio 10000688.gpio: registering 1 irq handlers
```

Reply:
Fixed.